extensions/wikia/UserProfilePageV3/templates/UserProfilePage_renderUserIdentityBox.php

```diff
@@ -189,6 +189,14 @@
 						<li><?= wfMessage( 'user-identity-box-zero-state-gender' )->escaped(); ?></li>
 					<? endif; ?>
 				<? endif; ?>
+				<? if ( !empty( $user['bio'] ) ): ?>
+					<li id="discussion-bio"><?= wfMessage( 'user-identity-bio', $user['bio'] )->plain(); ?></li>
```

Comment:
Sounds good. Updated: https://github.com/Wikia/app/pull/12337/commits/335299359936d40e6761c42eeedd43cb5be03b02. Thanks for looking this over 👍